### DISCUSSION TITLE
Please update minimum ios target

### DISCUSSION THREAD

**Discussion by vulnfincorp** [luicfrr/react-native-vision-camera-face-detector#60] [General]
I use expo sdk51 => podfile:  platform :ios,'13.4'
pod install => error "Specs satisfying the `VisionCameraFaceDetector (from `../node_modules/react-native-vision-camera-face-detector`)` dependency were found, but they required a higher minimum deployment target"

Please update minimum ios target to '13.4'

**Comment by luicfrr** [comment]
Supporting IOS versions older than 14 is not a feature I'm planning to.

You can create a local patch in your app so it will support it.
Do it at your own risk and think twice before opening issues.
